src/components/Navbar/Navbar.tsx

```diff
@@ -13,7 +13,7 @@ const Navbar = () => {
         <TimerNav />
       </div>
       <WelcomeBoard />
-      <div className="navBarButtons">
+      <div className="resetAndHTPBtns">
```

Comment:
changed class name so I could shrink this div a bit, to make the navbar more centered